site/src/pages/index.astro

```diff
@@ -33,77 +32,190 @@ import MetaLayout from "@/layouts/MetaLayout.astro";
         </dd>
+        <dt>3.3.1 Error Identification</dt>
+        <dd>
+          Empty search results and misspellings do not throw an error.(**Do others agree?)
+        </dd>   
       </dl>
-    </MetaFailureSection>
 
-    <MetaFailureSection href="museum/" title="Home">
-      <dl slot="wcag2">
-        <dt>1.3.2: Meaningful Sequence</dt>
+      <dl slot="wcag3">
+        <dt>Order of content</dt>
         <dd>
-          The overline text over each title in the carousel uses space characters instead of letter-spacing,
-          changing their interpretation.
+          The overline text over each title in the carousel uses space characters between each letter. 
+        </dd>       
+        <dt>Title<dt>
+        <dd>The search result page title is not accurate</dd>
+        <dt>Input labels</dt>
+        <dd>The search field has no label</dd>
+        <dt>Notification of Change</dt>
+        <dd>No programmatic notification occurs when the content changes</dd>
+        <dt>Name, Role, Value, State</dt>
+        <dd>
+          There is no search landmark indicating the presence or purpose of the search box and no label on the search field.
         </dd>
-        <dt>1.4.6: Contrast (Enhanced)</dt>
+        <dt>Error Identification</dt>
+        <dd>
+          Empty search results and misspellings do not throw an error.(**Do others agree?)
+        </dd> 
+        <dt>Text size</dt>
+        <dd>The text size in the footer is lower contrast and tiny</dd>    
+        <dt>Maximum text contrast</dt>
+        <dd>
+        The title in the header is 21:1 - is this above the contrast maximum?</dd>
+        <dt>Minimum text contrast</dt>
         <dd>
           The scrim on the carousel is not always enough to guarantee sufficient contrast.
         </dd>
-        <dt>2.2.2: Pause, Stop, Hide</dt>
+        <dt>Motion</dt>
         <dd>
           The carousel does not include controls to stop moving and updating.
         </dd>
+        <dt>Name, role, value, state (Needs Discussion)</dt>
+        <dd>
+          The Read More and Explore link text does not provide enough information to know which links go where.
+        </dd> 
+        <dt>Text size</dt>
+        <dd>The text size in the footer is a lower contrast and smaller size.</dd>                   
       </dl>
     </MetaFailureSection>
 
-    <MetaFailureSection href="museum/collections/" title="Collections">
+    <MetaFailureSection href="museum/collections/" title="Collections Page">
+      <p>This section includes only the collection page.</p>
+
+      <p>This section demonstrates content that has a moderate-to-high number of accessibilty failures, some of which would be very diffcult to work around. The failures present a mix of those that can be found through automated testing and those that require manual testing.</p>
+
       <dl slot="wcag2">
+        <dt>1.1.1: Non-text Content</dt>
+        <dd>
+          The alt text on some images is not accurate
```

Comment:
Noting for future that this particular note for `museum/collections/` seems misplaced. The images in the collections list intentionally have `alt=""`, and other images in the header/footer should be fine.